set.mm

```diff
@@ -757432,6 +757435,319 @@ last digit of the integer part (for ` b = ` 10 the first digit before
+  ("Multivariate functions",   19-May-2024,
+  ~ https://en.wikipedia.org/wiki/Function_(mathematics)#Multivariate_functions
+  ): "A _multivariate function_, multivariable function, or function of several
+  variables is a function that depends on several arguments.  ...  Formally, a
+  function of n variables is a function whose domain is a set of n-tuples.
+  For example, multiplication of integers is a function of two variables, or
+  _bivariate function_, whose domain is the set of all ordered pairs (2-tuples)
+  of integers, and whose codomain is the set of integers.  The same is true for
+  every binary operation.  Commonly, an n-tuple is denoted enclosed between
+  parentheses, such as in ( 1 , 2 , ... , n ).  When using functional notation,
+  one usually omits the parentheses surrounding tuples, writing
+  f ( x_1 , ... , x_n ) instead of f ( ( x_1 , ... , x_n ) ).  Given n sets
+  X_1 , ... , X_n , the set of all n-tuples ( x_1 , ... , x_n ) such that
+  x_1 is element of X_1 , ... , x_n is element of X_n is called the Cartesian
+  product of X_1 , ... , X_n , and denoted X_1 X ... X X_n .
+  Therefore, a multivariate function is a function that has a Cartesian product
+  or a proper subset of a Cartesian product as a domain: ` f : U --> Y ` where
+  where the domain ` U ` has the form
+  ` U C_ ( ( ... ( ( X `` 1 ) X. ( X `` 2 ) ) X. ... ) X. ( X `` n ) ) `."
+
+  In the following, n-ary functions are defined as mappings (see ~ df-map )
+  from a finite sequence of arguments, which themselves are defined as
+  mappings from the half-open range of nonnegative integers to the domain of
+  each argument.  Furthermore, the definition is restricted to endofunctions,
+  meaning that the domain(s) of the argument(s) is identical with its codomain.
+  This means that the domains of all arguments are identical (in contrast to
+  the definition in Wikipedia, see above: here, we have
+  X_1 = X_2 = ... = X_n = X).
+
+  For small n, n-ary functions correspond to "usual" functions with a different
+  number of arguments:
+
+  - n = 0 (nullary functions): These correspond actually to constants,
+  see ~ 0aryfvalelfv and ~ mapsn : ` ( X ^m { (/) } ) `
+
+  - n = 1 (unary functions): These correspond actually to usual endofunctions,
+  see ~ naryenef and ~ efmndbas : ` ( X ^m X ) `
+
+  - n = 2 (binary functions): These correspond to usual operations on two
+  elements of the same set, compare with ~ df-intop : ` ( X ^m ( X X. X ) ) `
+
+  Instead of using indexed arguments (represented by a mapping as described
+  above), elements of Cartesian exponentiations ` ( U ^^ N ) ` (see
+  ~ df-finxp ) could have been used to represent multiple arguments.  However,
+  this concept is not fully developed yet (it is within a mathbox), and it is
+  currently based on ordinal numbers, e.g., ` ( U ^^ 2o ) `, instead of
+  integers, e.g., ` ( U ^^ 2 ) `, which is not very practical.
+
+$)
+
+  $c NaryF $.  $( Letter I for the n-ary functions. $)
```

Comment:
I don't understand wher "Letter I" comes from. Copy-paste or typo?

Reply:
Yes, this was a copy&paste error - fixed.